src/carousel.css.ts

```diff
@@ -0,0 +1,334 @@
+import {createVar, globalStyle, style} from '@vanilla-extract/css';
+import {vars as skinVars} from './skins/skin-contract.css';
+import * as mq from './media-queries.css';
+import {applyAlpha} from './utils/color';
+import {sprinkles} from './sprinkles.css';
+import {vars as responsiveLayoutVars} from './responsive-layout.css';
+import {desktopMediumColumn, desktopSmallColumn} from './grid-layout.css';
+
+const bulletBase = style([
+    sprinkles({
+        width: 8,
+        height: 8,
+        borderRadius: '50%',
+    }),
+    {
+        transition: 'transform 0.3s ease-in-out, background-color 0.3s ease-in-out',
+        zIndex: 2, // needed because images has zIndex 1, otherwise this component won't be shown
+
+        '@media': {
+            [mq.tabletOrSmaller]: {
+                width: 4,
+                height: 4,
+            },
+        },
+    },
```

Comment:
I dont know why, but in my PRs im using always mobile/tablet as default, and isDesktop to media queries, I dont know either if this has any sense xD